gambit/1.0.0/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# Software installation, no database files
+FROM mambaorg/micromamba:0.27.0 as app_base
+
+ARG GAMBIT_SOFTWARE_VERSION="1.0.0"
+ARG GAMBIT_GIT_TAG=v${GAMBIT_SOFTWARE_VERSION}
+ARG GAMBIT_SRC_URL=https://github.com/jlumpe/gambit/archive/refs/tags/${GAMBIT_GIT_TAG}.tar.gz
+
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="GAMBIT"
+LABEL software.version=${GAMBIT_SOFTWARE_VERSION}
+LABEL description="Rapid genomic-distance comparison for taxonomic identification of microbial pathogens"
+LABEL website="https://github.com/jlumpe/gambit"
+LABEL license="https://github.com/jlumpe/gambit/blob/master/LICENSE"
+LABEL maintainer1="Kevin Libuit"
+LABEL maintainer.email1="kevin.libuit@theiagen.com"
+LABEL maintainer2="Michelle Scribner"
+LABEL maintainer.email2="michelle.scribner@theiagen.com"
+LABEL maintainer3="Jared Lumpe"
+LABEL maintainer.email3="jared@jaredlumpe.com"
+
+# Environment
+ENV GAMBIT_DB_PATH=/gambit-db
+ENV LC_ALL=C.UTF-8
+
+# Install mamba environment
+COPY --chown=$MAMBA_USER:$MAMBA_USER env.yaml /tmp/env.yaml
+RUN micromamba install -y -n base -f /tmp/env.yaml && \
+    micromamba clean --all --yes
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1  # Subsequent RUN commands use environment
+
+# Install GAMBIT package
+RUN pip install ${GAMBIT_SRC_URL} && \
+  micromamba clean -a -y
+
+USER root
+RUN mkdir $GAMBIT_DB_PATH /data && \
+  chown $MAMBA_USER:$MAMBA_USER $GAMBIT_DB_PATH /data
+USER $MAMBA_USER
+WORKDIR /data
+
+
+# With database files added
+FROM app_base AS app
+
+ARG GAMBIT_DB_BASE_URL=https://storage.googleapis.com/jlumpe-gambit/public/databases/refseq-curated/1.0
+ARG GAMBIT_DB_GENOMES_URL=$GAMBIT_DB_BASE_URL/gambit-refseq-curated-1.0.gdb
+ARG GAMBIT_DB_SIGNATURES_URL=$GAMBIT_DB_BASE_URL/gambit-refseq-curated-1.0.gs
+
+ADD --chown=$MAMBA_USER:$MAMBA_USER $GAMBIT_DB_GENOMES_URL $GAMBIT_DB_PATH/
+ADD --chown=$MAMBA_USER:$MAMBA_USER $GAMBIT_DB_SIGNATURES_URL $GAMBIT_DB_PATH/
+
+
```

Comment:
On line 54 could you please hardcode the conda/micromamba/mamba environment into the PATH variable? When I tried to use the docker image (in it's current state, dockerfile used to build image today) with my WDL workflow, it could not find the `gambit` executable nor other tools in the environment like `python3`

Code to add to dockerfile:
```Dockerfile
# adding micromamba environment to PATH so that the environment is always available in final app layer
ENV PATH=/opt/conda/bin:${PATH}
```

This will ensure that no matter how the docker container is instatiated and used, `gambit` will permanently be available to all users via the PATH variable. 

I take a similar approach with other tools, example: https://github.com/StaPH-B/docker-builds/blob/6b9fd6642836c9f5cf66b3dac47ce412ac9cb281/pangolin/4.1.3/Dockerfile#L74

Reply:
Done